exercises/practice/matrix/.meta/additional-tests.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "uuid": "9ca9c869-7220-4783-93ba-3e4f96b81732",
+        "description": "cannot extract row with no corresponding row in matrix",
+        "property": "row",
+        "input": {
+            "string": "1 2 3\n4 5 6\n7 8 9",
+            "index": 4
+        },
+        "expected": null,
+        "comments": [
+            "Additional test to ensure the method can handle invalid input"
+        ]
```

Comment:
I'd like this comment to answer two questions:
- Why was the test case not upstreamed / why do other languages not need this test?
- What additional value does the test provide?

suggestion:
```json
[
  "Additional test to ensure the method can handle invalid input",
  "Upstream does not want to add this test to avoid every exercise being about input validation.",
  "Rust puts a lot of emphasis on error handling, hence the idiomatic Option return type.",
  "With Option in the API, it makes sense to test for None at least once."
]
```

Reply:
That makes sense and good to know for the future!

I added the comments into both. We should be good to go now.